### PR TITLE
[t124509] update stock_picking_send_by_mail

### PR DESCRIPTION
**Pull request by ghost**
<div> Links to Odoo: </div> <ul>
<li><a target="_blank" href="https://odoo.braintec-group.com/web#view_type=form&model=project.task&id=124509">[t124509] MIUS V13 - Check Status stock_picking_send_by_mail</a></li>
</ul>
<div>Affected Modules:</div>
<table><thead><tr><th>Module</th><th>Ext</th></tr></thead>
<tr><td>stock_picking_send_by_mail</td><td>.xml, .py, .po</td></tr>
</tbody></table>

### CODE REVIEW

File: stock_picking_send_by_mail/data/stock_picking_mail.xml [hunk 1]
```diff
@@ -26,7 +20,7 @@
     <p style="border-left: 1px solid #8e0000; margin-left: 30px;">
        &nbsp;&nbsp;<strong>REFERENCES</strong><br />
        &nbsp;&nbsp;Picking number: <strong>${object.name}</strong><br />
-       &nbsp;&nbsp;Picking date: ${format_datetime(object.date, tz=user.tz, lang_code=object.partner_id.lang)} <br />
+       &nbsp;&nbsp;Picking date: ${format_datetime(object.date, tz=user.tz)} <br />
```
Comment (BT-rmartin) [inline stock_picking_send_by_mail/data/stock_picking_mail.xml:23]: You are removing lang_code=object.partner_id.lang when in the translation files keeps there

File: stock_picking_send_by_mail/data/stock_picking_mail.xml [hunk 2]
```diff
@@ -1,22 +1,16 @@
...
-        <field name="report_template" ref="stock.action_report_picking" />
-        <field name="report_name">${(object.name or '').replace('/','_')}</field>
-        <field name="lang">${object.partner_id.lang}</field>
-        <field
-            name="body_html"
-        ><![CDATA[
+
+        <record id="email_template_stock_picking" model="mail.template">
+            <field name="name">Stock Picking - Send by Email</field>
+            <field name="email_from">${(ctx['user_id'].email or '')|safe}</field>
```
Comment (BT-rmartin) [inline stock_picking_send_by_mail/data/stock_picking_mail.xml:5]: Why are you changing this email_from content?

File: stock_picking_send_by_mail/models/stock_picking.py [hunk 3]
```diff
@@ -6,30 +6,28 @@
 
 
 class StockPicking(models.Model):
-    _inherit = "stock.picking"
+    _inherit = 'stock.picking'
```
Comment (BT-rmartin) [inline stock_picking_send_by_mail/models/stock_picking.py:9]: Why changing to single quote? 
OCA repos use pre-commit with some prettyfier and so on that requires double quote. 
Don't change it or assure that the pre-commit works when working with OCA

File: stock_picking_send_by_mail/models/stock_picking.py [hunk 4]
```diff
@@ -6,30 +6,28 @@
...
 
     def action_picking_send(self):
         self.ensure_one()
-        template = self.env.ref(
-            "stock_picking_send_by_mail.email_template_stock_picking", False,
-        )
-        compose_form = self.env.ref("mail.email_compose_message_wizard_form", False,)
+        template = self.env.ref('stock_picking_send_by_mail.email_template_stock_picking', False)
```
Comment (BT-rmartin) [inline stock_picking_send_by_mail/models/stock_picking.py:13]: Don't do format changes unless necessary. 
OCA has just 79 characters, not 115 like us

File: stock_picking_send_by_mail/tests/test_stock_picking_send_by_mail.py [hunk 5]
```diff
@@ -2,35 +2,31 @@
...
-                        {
-                            "name": cls.product.name,
-                            "product_id": cls.product.id,
-                            "product_uom": cls.product.uom_id.id,
-                        },
-                    )
-                ],
-            }
-        )
+        cls.product = cls.env['product.product'].create({
```
Comment (BT-rmartin) [inline stock_picking_send_by_mail/tests/test_stock_picking_send_by_mail.py:13]: Again format and single quote unnecessary changes

**Review by BT-rmartin — CHANGES_REQUESTED** [review]
It's not so clear to me what you are adding beyond the LICENSE file and the tagged for the tests. 
There are also some changes in a mail template that I don't understand. 

This module is already merged by OCA in 13.0 so that means it passes all tests and that might be unnecessary.
Apart from that, if you would be really doing any contribution, you have to send it from your personal fork to OCA. 
Inside BT, you cannot do a PR against a main branch, as we shouldn't merge there because that branch is synced nightly against OCA. Instead, you should merge into a project branch. But in fact, in Mind we don't point to a project branch https://github.com/brain-tec/minditp/blob/13.0/.gitmodules#L20 but to 13.0, so if the changes would be really needed, you should create a project branch and change the reference and the entry in .gitmodules to point to it. 

Please, rethink if any of the changes of this PR is needed. If they are needed, you have to switch to a project branch and also contribute to OCA from your personal fork.

**Comment by ghost** [comment]
@BT-rmartin there were some migrated things done by team 2 in this module, including solving an issue with the tests: https://github.com/brain-tec/migration_port/issues/77, my doubt here is whether we're going to use what we already have forked from OCA or if I have to create a PR to the OCA repo to contribute with this.

**Comment by BT-rmartin** [comment]
@BT-aesteves Are those migrated things still issues? Did you run the tests? Are they needed? If so, you have to switch to a project branch and contribute to OCA, but if it's just about adding the tagged in the tests, I'm afraid OCA is not interested on that, as there the tests are already passing

**Comment by ghost** [comment]
@BT-rmartin I ran the tests without the migrations team 2 did and it works fine anyways, there's nothing needed from that migration  so i'm going to close this PR.
